### ISSUE TITLE
Results on ImageNet

### ISSUE THREAD

**Issue by ggjy** [FlyEgle/CMT-pytorch#1]
Hi，关于结果的一些训练方法，除了CMT-T(600e-1000e)和CMT-XS(350e-400e)要高于300e，当时CMT-T为了对标EfficientNet的训练，300epoch达不到79，600e以上才能到78以上，大概参数如下，其他的应该和论文差不多，比如一模一样的FLOPs的话，R=3.8其实是R=3.77这种，感觉无关紧要，就不贴在issue里了，投稿体验极差，本来想放代码的，也拖着了==，希望这些参数对你有帮助。

CMT-Tiny (600e-1000e is better) Top-1: 79.2
```python -m torch.distributed.launch --nproc_per_node=8 train_deit.py --model cmt_tiny --batch-size 256 --apex-amp --input-size 160 --weight-decay 0.04 --drop-path 0.1 --epochs 1000 --warmup-lr 1e-7 --warmup-epochs 20 --lr 8e-4 --min-lr 2e-5 --no-model-ema```

CMT-XS (350e-400e is better) Top-1: 81.8
```python -m torch.distributed.launch --nproc_per_node=8 train_deit.py --model cmt_extra_small --batch-size 128 --apex-amp --input-size 192 --weight-decay 0.08 --drop-path 0.1 --epochs 400 --warmup-epochs 20 --lr 7e-4 --min-lr 2e-5 --model-ema-decay 0.9998```

CMT-Small (300e) Top-1: 83.5
```python -m torch.distributed.launch --nproc_per_node=8 train_deit.py --model cmt_small --batch-size 128 --apex-amp --input-size 224 --weight-decay 0.05 --drop-path 0.1 --epochs 300 --model-ema-decay 0.99996```

CMT-Base (300e, FC Drop=0.3) Top-1: 84.5
```python -m torch.distributed.launch --nproc_per_node=8 train_deit.py --model cmt_base --batch-size 128 --apex-amp --input-size 256 --weight-decay 0.05 --drop-path 0.25 --epochs 300 --min-lr 2e-5 --model-ema-decay 0.99996```

CMT-Large (300e, FC Drop=0.3) Top-1: 84.8
```python -m torch.distributed.launch --nproc_per_node=8 train_deit.py --model cmt_large --batch-size 128 --apex-amp --input-size 288 --weight-decay 0.05 --drop-path 0.4 --epochs 300 --model-ema-decay 0.99996```

**Comment by ggjy** [comment]
Besides, CMT-Base needs 2 nodes, 8 V100 per node, CMT-Large needs 4 nodes. 8V100 per node.

**Comment by ggjy** [comment]
I use the DeiT codebase, so the acutal Learning Rate is:
```args.lr = args.lr * args.batch_size * utils.get_world_size() / 512.0```

**Comment by FlyEgle** [comment]
哇，感谢感谢，后面我按照这个来试试，不过CMT的FLOPs小，但是速度并不快，慢很多其实，这个问题以后会解决吗？

**Comment by ggjy** [comment]
一个是速度慢，主要是depth-wise在GPU慢以及attention，另一个是太吃显存。这两个是打算解决的问题。

**Comment by developer0hye** [comment]
@ggjy 

Hi, ggjy!

Could you have a plan to upload the official CMT code?
